aspnetcore/blazor/lifecycle.md

```diff
@@ -0,0 +1,173 @@
+---
+title: ASP.NET Core Blazor lifecycle
+author: guardrex
+description: Learn how to use Razor component lifecycle methods in ASP.NET Core Blazor apps.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 11/23/2019
+no-loc: [Blazor]
+uid: blazor/lifecycle
+---
+# ASP.NET Core Blazor lifecycle
+
+By [Luke Latham](https://github.com/guardrex) and [Daniel Roth](https://github.com/danroth27)
+
+The Blazor framework includes synchronous and asynchronous lifecycle methods. Override lifecycle methods to perform additional operations on components during component initialization and rendering.
+
+### Component initialization methods
```

Comment:
Is it deliberate that these aren't top level headings? Currently these headings don't show up in the summary view.

Reply:
Ah :eye: thanks ... I accidentally killed a heading for these when I was deleting a paragraph: There should be a *Lifecycle methods* H2 here to cover these H3s. I'll add that H2 back.